src/forms/textInput.test.jsx

```diff
@@ -1,129 +1,117 @@
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
-import { TestWrapper } from '../utils/testUtils';
 import TextInput from './TextInput';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
 
 describe('TextInput', function() {
-	const onChange = jest.fn();
+
 	const LABEL_TEXT = 'Super Hero',
 		VALUE = 'Batman',
 		NAME_ATTR = 'superhero',
-		MAX_LEN = '20',
+		MAX_LEN = 20,
 		ERROR_TEXT = 'Too wimpy.';
 
-	let textInputComponent,
-		inputEl;
-
-	beforeEach(() => {
-		const formAttrs = {
-			id: NAME_ATTR,
-			maxLength: MAX_LEN,
-			error: ERROR_TEXT,
-			required: true,
-		};
-		textInputComponent = TestUtils.renderIntoDocument(
-			<TestWrapper>
-				<TextInput
-					name={NAME_ATTR}
-					label={LABEL_TEXT}
-					value={VALUE}
-					{...formAttrs}
-				/>
-			</TestWrapper>
+	const formAttrs = {
+		id: NAME_ATTR,
+		maxLength: MAX_LEN,
+		required: true,
+		error: ERROR_TEXT,
+	};
+
+
+	it('creates a required input with given label, name, id and value ', () => {
+		const component = shallow(
+			<TextInput
+				name={NAME_ATTR}
+				label={LABEL_TEXT}
+				value={VALUE}
+				id={NAME_ATTR}
+				required
+			/>
 		);
 
-		inputEl = TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'input');
+		expect(toJson(component)).toMatchSnapshot();
```

Comment:
Just fyi I setup on a jest configuration to default to the `toJson()` snapshot serializer in `mup-web`: https://github.com/meetup/mup-web/pull/1291#discussion_r126980339

If we decide to use it, might be nice to have it in this repo as well.